testnet-contracts/package.json

```diff
@@ -24,12 +28,14 @@
     "chai-bignumber": "^3.0.0",
     "openzeppelin-solidity": "^2.1.3",
     "solc": "^0.5.0",
-    "web3": "^1.0.0-beta.52",
+    "web3": "^1.2.1",
     "web3-utils": "^1.0.0-beta.52"
   },
   "scripts": {
     "develop": "truffle develop",
-    "test": "truffle test",
-    "exit": ".exit"
+    "migrate": "truffle compile && truffle migrate --reset --all",
+    "peggy:address": "truffle exec scripts/getDeployedAddress.js",
+    "peggy:lock": "truffle exec scripts/sendLockTx.js",
```

Comment:
also needs --network develop

also, this script takes 30 seconds, super long, any idea why?

Reply:
Not sure. The `console.log(params)` executes almost immediately yet exec doesn't exit the script for a long time even though the only line remaining is the return statement.